product_quick_bom/product.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#   Copyright (C) 2015 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    bom_line_ids = fields.One2many(
+        'mrp.bom.line',
+        'product_tmpl_id',
+        string='Bom Line',
+        help='If you product is manufactured you can select'
+             'here the componant to product them')
+
+    @api.model
+    def _extract_bom_line(self, vals):
+        return vals.pop('bom_line_ids', {})
+
+    @api.multi
+    def _prepare_bom_vals(self, vals):
+        return {
+            'product_tmpl_id': self.id,
```

Comment:
Add `self.ensure_one()` as first line of the method to ensure you have only one record in self.